### PR TITLE
86b2j5p7n plantuml

### PR DESCRIPTION
**Pull request by AynulAmna**
I have updated Activity Diagram .

### CODE REVIEW

File: docs/schema/er_digram.wsd [hunk 1]
```diff
@@ -0,0 +1,71 @@
+@startuml
```
Comment (bsaranga) [inline docs/schema/er_digram.wsd:1]: Specify diagram title after @startuml like this:
`@startuml ER Diagram`

File: docs/uml/activity.wsd
Comment (bsaranga) [inline docs/uml/activity.wsd:1]: The activity diagram is too large for a single viewing, can we break it up into scenarios, such as:

1. Login scenario
2. Sign up scenario
3. Magic Link Signup/Signin scenario
4. Reservation scenario
5. etc...

**Review by bsaranga — CHANGES_REQUESTED** [review]

File: docs/uml/activity.wsd
Comment (AynulAmna) [reply docs/uml/activity.wsd:1]: For each scenario need to create separate file.?

Comment (bsaranga) [reply docs/uml/activity.wsd:1]: Yeah? If it's difficult to split a scenario keep it in one file, but let's try to divide them to scenarios as much as possible. Each scenario will be a separate file.

**Pull request closed without merging**